README.md

```diff
@@ -2,10 +2,9 @@
 
 ```shell
 pip install pre-commit detect-secrets
-mkdir .gds
-detect-secrets scan > .gds/.secrets.baseline
-detect-secrets audit .gds/.secrets.baseline
-pre-commit install -c .gds/.pre-commit-config.yaml #this file is from this repo
+mkdir ~/gds
+git clone https://github.com/alphagov/gds-pre-commit.git ~/gds/gds-pre-commit
+git config --global core.hooksPath $HOME/gds/gds-pre-commit/global_install/hooks
```

Comment:
Also what if $HOME/gds/gds-pre-commit is where the user has checked out that code to already?